src/styles/_statusbar.scss

```diff
@@ -14,7 +14,7 @@
   &.h5p-interactive-book-status-header {
     @include prefix(position, sticky, webkit);
     pointer-events: none;
-    background: transparent;
+    background: white;
```

Comment:
On the one hand named color values should not be used on production. Their actual appearance may vary from device to device (see https://developer.mozilla.org/en-US/docs/Web/CSS/color_value).

But first and foremost, setting it to a fixed value will cause trouble when using a "dark" mode or similar. Should probably rather be `var(--h5p-theme-ui-base)` like for the sidebar.